h/views/api/decorators/response.py

```diff
@@ -4,17 +4,25 @@
 from __future__ import unicode_literals
 
 from h.views.api import API_VERSION_DEFAULT
-from h.views.api.helpers.media_types import media_type_for_version
+from h.views.api.helpers.media_types import media_type_for_version, version_media_types
 
 
 def version_media_type_header(wrapped):
     """View decorator to add response header indicating API version"""
 
     def wrapper(context, request):
         response = wrapped(context, request)
-        # TODO: expand with logic for multiple versions once the app knows
-        # about multiple versions
+        # Assume default version...
         version_media_type = media_type_for_version(API_VERSION_DEFAULT)
+        # ...Unless we can determine otherwise from the Accept header
+        if request.accept:
+            version_accepts = [t for t in request.accept if t in version_media_types()]
+            if any(version_accepts):
```

Comment:
Could this be just `if version_accepts` since we only need to test for a non-empty list here.